verification/cbmc/proofs/aws_array_list_init_static/aws_array_list_init_static_harness.c

```diff
@@ -25,6 +25,7 @@ void aws_array_list_init_static_harness() {
 
     /* perform operation under verification */
     uint8_t *raw_array = bounded_malloc(len);
+    __CPROVER_assume(raw_array != NULL);
```

Comment:
What about the `len == 0, ptr == NULL` case

Reply:
This maps to an assumption in the function under verification and `len` is not a input in this function.
```c
AWS_STATIC_IMPL
void aws_array_list_init_static(
    struct aws_array_list *AWS_RESTRICT list,
    void *raw_array,
    size_t item_count,
    size_t item_size) {

    AWS_FATAL_PRECONDITION(list != NULL);
    AWS_FATAL_PRECONDITION(raw_array != NULL);
    AWS_FATAL_PRECONDITION(item_count > 0);
    AWS_FATAL_PRECONDITION(item_size > 0);
```